staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/webhook.go

```diff
@@ -207,6 +210,25 @@ func (a *Webhook) ShouldCallHook(h webhook.WebhookAccessor, attr admission.Attri
 		return nil, matchObjErr
 	}
 
+	matchConditions := h.GetMatchConditions()
+	expressions := make([]cel.ExpressionAccessor, len(matchConditions))
+	for i, matchCondition := range matchConditions {
+		expressions[i] = &MatchCondition{
+			Name: matchCondition.Name,
+			Expression: matchCondition.Expression,
+		}
```

Comment:
It doesn't like me defining the GetExpression() method on it after making it an alias. I could define the GetExpression method so that it implements `var _ cel.ExpressionAccessor = &MatchCondition{}` within admissionregistration/v1 though but I don't think that follows the existing pattern. Open to any suggestions for best way to handle this.

Reply:
Ah sorry, I misused the term type alias. I'm not sure what the correct term for this is...

What I meant was to define a local MatchCondition type as just being a `v1.MatchCondition`. In other words, I think the following should work:

```go
type MatchCondition v1.MatchCondition

func (m *MatchCondition) GetExpression() string {
	return m.Expression
}

func (m *MatchCondition) ReturnTypes() []*cel.Type {
	return []*cel.Type{cel.BoolType}
}

func (a *Webhook) ShouldCallHook(...) ... {
	// ...

	matchConditions := h.GetMatchConditions()
	expressions := make([]cel.ExpressionAccessor, len(matchConditions))
	for i, matchCondition := range matchConditions {
		expressions[i] = cel.ExpressionAccessor(matchCondition)
	}

}
```